apps/src/templates/progress/progressHelpers.js

```diff
@@ -2,12 +2,28 @@ import { ViewType } from '@cdo/apps/code-studio/stageLockRedux';
 import { isHiddenForSection } from '@cdo/apps/code-studio/hiddenStageRedux';
 
 /**
- * Used by components to determine whether a particular lesson is hidden or not.
- * viewAs, sectionId, and hiddenStageState all represent state extracted from
- * our redux store.
- * @returns {boolean} True if the provided lesson is hidden
+ * This is conceptually similar to being a selector, except that it operates on
+ * the entire store state. It is used by components to determine whether a
+ * particular lesson is visible, or hidden entirely.
+ * @param {number} lessonid - id of the lesson we're querying
```

Comment:
Incorrect param doc - looks like you now take the whole lesson object, not just the id.